Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/unfettered jobs.txt

	on complete
		payment 7000 2000
		dialog phrase "unfettered tribute payment dialog"
	source
		attributes "unfettered"
	stopover
		distance 4 5
		attributes "hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic cargo on visit"
	on stopover
		dialog phrase "unfettered tribute pickup dialog"

mission "Unfettered Tribute 3"
	name "Hai Tribute to <planet>"
	job
	repeat
	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
	to offer
		has "Unfettered Jump Drive 2: offered"
		random < 20
	cargo "food (tribute)" 45 2 .05
	on complete
		payment 9000 2400
		dialog phrase "unfettered tribute payment dialog"
	source
		attributes "unfettered"
	stopover
		distance 5 6
		attributes "hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic cargo on visit"
	on stopover
		dialog phrase "unfettered tribute pickup dialog"

mission "Proud Unfettered Boarded"
	boarding
	repeat
	minor
	to offer
		"flagship crew" > 120
		random < "flagship crew" / 8
	source
		government "Hai (Unfettered)"
		category "Heavy Warship"
	on offer
		conversation

> 			`As you board the <origin> you immediately notice something is wrong, the ship is starting to explode! The captain sends you a message. "This is not a fair fight. Better to die with my crew, standing, than risk being slaughtered by false, inferior, fettered humans! You will not be allowed to us this ship against our brothers and sisters, I will take you down with us!"`

Author: They never really referred to humans except for (maybe pirates) and alphas as far as I know of, this is really meant to be like an "insult" whilst staying ES friendly.

				flee
	destination "Darkcloak"

mission "Unfettered Silicon Asteroid Mining"
	name `Gather silicon from asteroids`
	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot and then bring them back to <destination> for <payment>.`
	job
	repeat
	source "Darkcloak"
	to offer
		has "First Contact: Unfettered: declined"
		random < 60
	on complete
		payment 250000
		dialog `The Unfettered unloading the materials you mined seem enthused. "This will allow us to repair our ships!" they tell you as they hand over your payment of <payment> credits.`
		outfit "Silicon" -50

mission "Unfettered Silver Asteroid Mining"
	name `Gather silver from asteroids`

> 	description `Mine and then collect 15 tons of Silver from the asteroid field in Ehma Ti and then bring them back to <destination> for <payment>.`

Reviewer: ```suggestion
	description `Mine and then collect 15 tons of silver from the asteroid field in Ehma Ti and then bring them back to <destination> for <payment>.`
```

	to offer
		has "Unfettered Jump Drive 1: offered"
		random < 80
	cargo "medicals" 15 2 .05
	stopover
		distance 5 7
		government "Hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic cargo on visit"
	on stopover
		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. "Tell our brothers and sisters to stay strong,"	they ask while loading the supplies on your ship.`
	on complete
		payment 10000 4000
		dialog `The local health specialists seem relieved that you have managed to bring the needed medical supplies back in time.`

mission "Unfettered Challenge[0]"
	name `Defeat the young captain`
	description `A young Unfettered captain has provoked you in a duel, disable his ship for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 2: offered"
		random < 60
	npc save disable
		government "Hai (Unfettered Challenge)"
		personality disables staying heroic target
		fleet
			names "hai"
			variant
				"Sea Scorpion (Pulse)"
		conversation
			action
				payment 200000
				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 15
				fail "Unfettered Challenge[0]"

> 			`You receive a message from the <npc> : "Raaaah... Well you have beaten me alright, so I'll pay up... Here are the <payment> credits, captain."`

Reviewer: Would it make sense for the Unfettered here to react this way? When you disable Unfettered ships in normal battle, their hails are extremely hostile to you. I wouldn't expect an Unfettered to be so calm after defeat, considering that. However, I haven't read through the mission file yet, so if there's something there that justifies this response (becoming respected with the Unfettered by the time this job becomes available to you, some aspect of Unfettered culture I haven't seen) then please let me know.

Author: Well he is very angry to have lost but you did just break his spirit by disabling his ship, this is a duel after all, the intention was to fight you, and maybe lose. I really do not want the player to get the impression he could kill or capture this ship without consequences, also the "duellists" do not use the same hails as the rest of the Unfettered when they are hostile (given the idea is they ask you to fight them), they are only annoyed, not provocative.

Author: That said, this one is the least experienced of all of the captains you fight, so I do agree he could be a bit more aggressive.
I tried to have them react different ways (not happy to "wise")

Author: I changed the first 2 dialogs from the captains, the first one is clearly angry now, second one is neutral and third one stays positive

				decline

mission "Unfettered Challenge[1]"
	name `Defeat the captain`

> 	description `An Unfettered captain has provoked you in a duel, disable his ship for <payment>.`

Reviewer: ```suggestion
	description `An Unfettered captain has provoked you in a duel. Disable his ship for <payment>.`
```

	job
	repeat
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 2: offered"
		has "Unfettered: Prove Your Strength"
		random < 40
	npc save disable
		government "Hai (Unfettered Challenge)"
		personality disables staying heroic target
		fleet
			names "hai"
			variant
				"Sea Scorpion (Pulse)" 2
		conversation
			action
				payment 400000
				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 30
				fail "Unfettered Challenge[1]"

> 			`You receive a message from the <npc> : "Well done captain! But, I will not be so easy to beat next time! Here is the agreed <payment> credits. You earned it."`

Reviewer: Same as above.

				decline

mission "Unfettered Challenge[2]"
	name `Defeat the experienced captain`

> 	description `An experienced Unfettered captain has provoked you in a duel, disable his ship for <payment>.`

Reviewer: ```suggestion
	description `An experienced Unfettered captain has provoked you into a duel. Disable his ship for <payment>.`
```

	job
	repeat
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 3: offered"
		has "Unfettered: Prove Your Strength"
		random < 10
	npc save disable
		government "Hai (Unfettered Challenge)"
		personality disables staying heroic target
		fleet
			names "hai"
			variant
				"Shield Beetle (Pulse)"
				"Sea Scorpion (Pulse)"
		dialog `You seem to learn from our encounters as well... Well next time it will be different.`
		conversation
			action
				payment 600000
				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 60
				fail "Unfettered Challenge[2]"

> 			`You receive a message from the <npc> : "You seem to learn from our encounters as well! Well, next time it will be different. Here are the agreed <payment> credits, captain. Hopefully it allows you to improve the <ship> before our next battle..."`

Reviewer: Same as above.

				decline

mission "Get Jump Drives(1)"
	name `Collect a jump drive`
	description `Go to the pirate world of <destination> and recover a jump drive from a vault for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	destination "Darkcloak"
	to offer

> 		random < 75

Reviewer: Would there be enough jump drives on human pirate worlds to justify having this job pop up so often? I do think that there would be a few jump drives on human pirate worlds, but certainly not enough to have a repeatable job for it. At most there could be a dedicated mission for it, where an Unfettered tips you off to a cache of Jump Drives some human warlord is hoarding.

Author: There is a scripted version of this, these jump drives are placed there by the Alphas for most worlds, except for the one next to Korath space where they get those from raiders.
I should indeed make it happen less often still.

Reviewer: Ok, that makes sense. The offer value should definitely be lower though.

Author: > Ok, that makes sense. The offer value should definitely be lower though.

How about 30%, 15% and 5% ? 
I mostly made it frequent from a gameplay perspective because it's quite a long way to go and get a jump drive (for the same reason you are payed 1M instead of less and having some of it go to the alphas)

Reviewer: That sounds good to me.

		has "Unfettered: Prove your strength 2: done"
	stopover
		distance 12 18
		government "Pirate"
	on stopover
		dialog `You manage to land and retrieve the jump drive without too much trouble. Do not rely on luck too long though, you had better depart quickly!`
		outfit "Jump Drive"
	on visit
		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
	on complete
		payment 1000000
		outfit "Jump Drive" -1
		dialog `The Unfettered are more than content with paying <payment> credits in exchange for your precious cargo.`

mission "Get Jump Drives(2)"
	name `Collect 2 jump drives`
	description `Go to the pirate world of <destination> and recover 2 jump drives from a vault for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	destination "Darkcloak"
	to offer

> 		random < 45

Reviewer: Same as above.

		has "Unfettered: Prove your strength 2: done"
	stopover
		distance 12 18
		government "Pirate"
	on stopover
		dialog `You manage to land and retrieve the jump drives without too much trouble. Do not rely on luck too long though, you had better depart quickly!`
		outfit "Jump Drive" 2
	on visit
		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
	on complete
		payment 2000000
		outfit "Jump Drive" -2
		dialog `The Unfettered are more than content with paying <payment> credits in exchange for your precious cargo.`

mission "Get Jump Drives(3)"
	name `Collect 3 jump drives`
	description `Go to the pirate world of <destination> and recover 3 jump drives from a vault for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	destination "Darkcloak"
	to offer

> 		random < 15

Reviewer: Same as above.

		has "Unfettered: Prove your strength 2: done"
	stopover "Deadman's Cove"
	on stopover
		dialog `You manage to land and retrieve the jump drives without too much trouble. Do not rely on luck too long though, you had better depart quickly!`
		outfit "Jump Drive" 3
	on visit
		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
	on complete
		payment 3000000
		outfit "Jump Drive" -3
		dialog `The Unfettered are more than content with paying <payment> credits in exchange for your precious cargo.`

mission "Bring True Hai"
	name `Bring True Hai`
	description `Land on <stopovers> and bring back <bunks> True Hai to where they belong, on <destination> for <payment>.`
	job
	repeat
	passengers 5 8 .2
	source
		attributes "unfettered"
	destination
		attributes "unfettered"
	to offer
		random < 75
		has "Unfettered: The Word Has Spread: done"
	stopover
		distance 3 6
		government "Hai"
		attributes "spaceport"
	on stopover
		dialog `You have picked up all <bunks> Hai, now all that is left is to bring them to <destination>.`
	on complete
		payment
		payment 150000

> 		dialog `You feel good that you managed to bring these Hai to their True home. The Unfettered hand you the agree payment of <payment>.`

Reviewer: Generally speaking, the game isn't supposed to tell the player how they feel about something. 
```suggestion
		dialog `You feel good that you managed to bring these Hai to their True home. The Unfettered hand you the agreed payment of <payment>.`
```

mission "Darkcloak Ship Materials"
	name `Ship materials from <stopovers>`

> 	description `The Shipyards of <origin> are in grave need of raw materials. Go to the pirate world of <destination> to get the materials for building new ships, and bring them back by <date> for <payment>.`

Reviewer 2: ```suggestion
	description `The shipyards of <origin> are in grave need of raw materials. Go to the pirate world of <stopovers> to get the materials for building new ships, and bring them back by <date> for <payment>.`
```

I think this is the right fix here.

Author: yup you're totally right, I don't always read those descriptions in game, nice catch!

	job
	repeat
	deadline 15 1
	source `Darkcloak`
	cargo "ship materials" 50 15 .015 
	to offer
		random < 50
		has "Unfettered Jump Drive 3: done"
		has "Unfettered: The War Has Started"
	stopover
		distance 12 18
		government "Pirate"
	on stopover

> 		dialog `After some searching you manage to find your contacts, a few shady individuals who seem especially anxious not to be noticed dealing with you. Together you find a quieter place to proceed with the exchange, and later find the materials at the agreed location. Now head back to <origin> to deliver them.`

Reviewer: ```suggestion
		dialog `After some searching you manage to find your contacts: a few shady individuals who seem especially anxious not to be noticed dealing with you. Together you find a quieter place to proceed with the exchange, and later find the materials at the agreed location. Now head back to <origin> to deliver them.`
```

	stopover
		distance 5 6
		attributes "hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic cargo on visit"
	on stopover
		dialog phrase "unfettered tribute pickup dialog"

mission "Proud Unfettered Boarded"
	boarding
	repeat
	minor
	to offer
		"flagship crew" > 120
		random < "flagship crew" / 8
	source
		government "Hai (Unfettered)"
		category "Heavy Warship"
	on offer
		conversation
			`As you board the <origin> you immediately notice something is wrong: the ship is starting to explode! The captain sends you a message. "This is not a fair fight. Better to die with my crew standing, than risk being slaughtered by inferior humans! You will not be allowed to capture this ship and use it against our brothers and sisters. I will take you down with us!"`
				flee
	destination "Darkcloak"

mission "Unfettered Silicon Asteroid Mining"
	name `Gather silicon from asteroids`
	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot and then bring them back to <destination> for <payment>.`
	job
	repeat
	source "Darkcloak"
	to offer
		has "First Contact: Unfettered: declined"
		random < 60
	on complete
		payment 250000

> 		dialog `The Unfettered unloading the materials you mined seem enthused. "This will allow us to repair our ships!" they tell you as they hand over your payment of <payment> credits.`

Author: You're completely right, I wasn't even sure I would add these missions at first because we don't have the option to ask for a random amount of outfit; like a mineral. But yeah it needs some different dialogs for the different amounts and better ones.
Please do be as critical as you can be!

	job
	repeat
	deadline 10 1
	source
		attributes "unfettered"
	destination "Darkcloak"
	to offer
		has "Unfettered Jump Drive 1: offered"
		random < 80
	cargo "medicals" 15 2 .05
	stopover
		distance 5 7
		government "Hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic cargo on visit"
	on stopover
		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. "Tell our brothers and sisters to stay strong,"	they ask while loading the supplies on your ship.`
	on complete
		payment 10000 4000
		dialog `The local health specialists seem relieved that you have managed to bring the needed medical supplies back in time.`

mission "Unfettered Challenge[0]"
	name `Defeat the young captain`
	description `A young Unfettered captain has provoked you in a duel. Disable his ship for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 2: offered"
		random < 60
	npc save disable
		government "Hai (Unfettered Challenge)"
		personality disables staying heroic target
		fleet
			names "hai"
			variant
				"Sea Scorpion (Pulse)"
		conversation
			action
				payment 200000

> 				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 15

Author: it is how much you will lose for disabling a crew of 15, but a bit above it to be sure
because if you do smth to a government even if said government doesn't care about it, others will look at it with their values (this gov ignores disabling but UHai gov doesn't)

				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 60
				fail "Unfettered Challenge[2]"
			`You receive a message from the <npc> : "Nicely done! You seem to learn from our encounters as well... But next time will be different, I assure you. For now, here are the agreed <payment> credits, captain. Hopefully it allows you to improve the <ship> before our next battle..."`
				decline

mission "Get Jump Drives(1)"
	name `Collect a jump drive`
	description `Go to the pirate world of <stopovers> and recover a jump drive from a vault, then bring to <destination> for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	destination "Darkcloak"
	to offer
		random < 30
		has "Unfettered: Prove your strength 2: done"
	stopover
		distance 12 18
		government "Pirate"
	on stopover
		dialog `You manage to land and retrieve the jump drive without too much trouble. Do not rely on luck too long though, you had better depart quickly!`
		outfit "Jump Drive"
	on visit
		dialog `You have a mission to bring a jump drive here. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`
	on complete

> 		payment 1000000

Author: Its already quite a long way to get there, and these missions are quite uncommon, I placed them at this price to make you sometimes want to take them given how much money tribute gives, or there is no point in this mission from a gameplay perspective
They really want their JD, too, but maybe I could make it be 800 000?

> 		payment 1000000

Reviewer 3: Maybe, I was thinking a bit less as you're traveling through mostly peaceful space and you could potentially be friendly with pirate which left just distance being the factor but player being friendly with all the factions at once is probably unlikely enough.

> 		payment 1000000

Author: You strike a hard bargain

For context, you get payed 1M for going into friendly Hai space taking tributes of 50 tons, easily.
This may take less cargo, but it will be into probable enemy territory, where you may have to pay a bribe only give 500k?

> 		payment 1000000

Reviewer 2: 50/50 for delivering it but not supplying it sounds about right.

At this point for the players, pirates should be an inconvenience more than a dire threat, as well.

That's a large enough distance, too, that smart players will make additional money moving commodities back and forth anyway.

	on complete
		payment 7000 2000
		dialog phrase "unfettered tribute payment dialog"
	source
		attributes "unfettered"
	stopover
		distance 4 5
		attributes "hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic missing stopover or cargo"
	on stopover
		dialog phrase "unfettered tribute pickup dialog"

mission "Unfettered Tribute 3"
	name "Hai Tribute to <planet>"
	job
	repeat
	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
	to offer
		has "Unfettered Jump Drive 2: offered"
		random < 40
	cargo "food (tribute)" 45 2 .05
	on complete
		payment 9000 2400
		dialog phrase "unfettered tribute payment dialog"
	source
		attributes "unfettered"
	stopover
		distance 5 6
		attributes "hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic missing stopover or cargo"
	on stopover
		dialog phrase "unfettered tribute pickup dialog"

mission "Proud Unfettered Boarded"
	boarding
	repeat
	minor
	to offer
		"flagship crew" > 120
		random < "flagship crew" / 8
	source
		government "Hai (Unfettered)"
		category "Heavy Warship"
	on offer
		conversation

> 			`As you board the <origin> you immediately notice something is wrong: the ship is starting to explode! The captain sends you a message. "This is not a fair fight. It is better to die with my crew standing than risk being slaughtered by inferior humans! You will not be allowed to capture this ship and use it against our brothers and sisters. I will take you down with us!"`

Author: I did that, hopefully I didn't destroy English too much in those phrase options.

Also sometimes I had an issue that I cant make it part of the conversation, so I had to cut it by a dialog, idk how that'll look but unless I do some code changes that's the only way to include a phrase in it.

				flee
	destination "Darkcloak"

mission "Unfettered Silicon Asteroid Mining"
	name `Gather silicon`
	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot, and then bring them back to <destination> for <payment>.`
	job
	repeat
	source "Darkcloak"
	to offer
		has "First Contact: Unfettered: declined"
		random < 60
	on complete
		payment 250000
		dialog `The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use those asteroids as target practice, you know, but most find it dull and too easy to fight an enemy that does not shoot or react in any other way," one tells you as it hands over your payment of <payment> credits.`
		outfit "Silicon" -50

mission "Unfettered Silver Asteroid Mining"
	name `Gather silver from asteroids`
	description `Mine and then collect 15 tons of silver from the asteroid field in Ehma Ti and then bring them back to <destination> for <payment>.`
	job
	repeat

> 	source "Firelode"

Author: If I remember correctly the silver is in the system of Firelode
Currently they ask you to mine in the same system you get the mission at

> 	source "Firelode"

Reviewer 4: So for the silicon mission: Offered on Darkcloak, mine in system Darkcloak is in, return to Darkcloak.
For the silver mission: Offered on Firelode, mine in system Firelode is in, return to Darkcloak.
Obviously that implies that most of the production is taking place on Darkcloak. Why wouldn't Darkcloak offer the same job since they're the ones that need the silver?

	job
	repeat
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 2: offered"
		has "Unfettered: Prove Your Strength"
		random < 40
	npc save disable
		government "Hai (Unfettered Challenge)"
		personality disables staying heroic target
		fleet
			names "hai"
			variant
				"Sea Scorpion (Pulse)" 2
		conversation
			action
				payment 400000
				"reputation: Hai (Unfettered)" += 30
				fail "Unfettered Challenge [1]"
			`You receive a message from the <npc> : "Raaaah... Well, you have beaten me fairly, so I'll pay up... Here are the <payment> credits, captain. Next time I will not underestimate you anymore!"`
				decline

mission "Unfettered Challenge [2]"
	name `Defeat experienced Unfettered captain`
	description `An experienced Unfettered captain has provoked you into a duel. Disable his ship for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 3: offered"
		has "Unfettered: Prove Your Strength"
		random < 10
	npc save disable
		government "Hai (Unfettered Challenge)"
		personality disables staying heroic target
		fleet
			names "hai"
			variant
				"Shield Beetle (Pulse)"
				"Sea Scorpion (Pulse)"

> 		dialog `You receive an ominous message from the <npc>: "You seem to learn from our encounters well... next time it will be different.`

Author: it was just an error I think

		conversation
			action
				payment 600000
				"reputation: Hai (Unfettered)" += 60
				fail "Unfettered Challenge [2]"
			`You receive a message from the <npc> : "Nicely done! You seem to learn from our encounters well... But next time will be different, I assure you. For now, here are the agreed <payment> credits, captain. Hopefully it allows you to improve the <ship> before our next battle..."`
				decline

phrase "jump drive or stopover"
	word
		`You have a mission to bring a jump drive here on <planet>, from <stopovers>. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`

> mission "Get Jump Drives(1)"

Author: The Unfettered dont want you to know how they get these JD

> mission "Get Jump Drives(1)"

Reviewer 4: But the player never gets a chance to ask about this. If you're incorporating some mystery in, you need to let the player investigate it, even if it results in a dead end. Make sure the player can ask the Unfettered "Where are these JDs coming from?" even if the Unfettered just tell them "You're not allowed to know that" or dodge the question. It also could be good to include some hints/signs about where they're coming from during the mission where we take jump drives in the campaign.
I'd suggest implementing these changes, then make these jobs available after that campaign mission.

> mission "Get Jump Drives(1)"

Author: in `"Unfettered: Prove your strength 2: done"` you can ask, and he does dodge the question

	source
		attributes "unfettered"
	stopover
		distance 5 6
		attributes "hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic missing stopover or cargo"
	on stopover
		dialog phrase "unfettered tribute pickup dialog"

phrase "unfettered boarded"
	word
		`"This is not a fair fight. It is better to die with my crew standing than risk being slaughtered by inferior humans! You will not be allowed to capture this ship and use it against our brothers and sisters. I will take you down with us!"`
		`"This ship is the property of Unfettered, Hai even, but never monkey! We'd rather die than let you have it."`
		`"It is truly sad it had to come to this, but we know you will not listen to reason. We have no choice but to destroy our ship."`

mission "Proud Unfettered Boarded"
	boarding
	repeat
	minor
	to offer
		"flagship crew" > 120
		random < "flagship crew" / 8
	source
		government "Hai (Unfettered)"
		category "Heavy Warship"
	on offer
		conversation
			`As you board the <origin> you immediately notice something is wrong: the ship is starting to explode! The captain sends you a message.`
		dialog phrase "unfettered boarded"
		conversation

> 			`It is no bluff... Best get out of here!`

Reviewer: ```suggestion
			`They weren't bluffing. Better get out of here quickly!`
```

				flee
	destination "Darkcloak"

phrase "unfettered mining payment dialog"
	word
		`The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use those asteroids as target practice, you know, but most find it dull and too easy to fight an enemy that does not shoot or react in any other way," one tells you as it hands over your payment of <payment> credits.`

> 		`The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships! Too few captains are patient enough to gather asteroids like you are. Most of them try to get the resources from enemy ships, but this is a safer way to do it." One tells you as it hands over your payment of <payment> credits.`

Reviewer: ```suggestion
		`The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships! Too few captains are patient enough to gather asteroids like you are. Most of them try to get the resources from enemy ships, but this is a safer way to do it," one tells you as it hands over your payment of <payment>.`
```

> 		`Unfettered unload the materials you mined, without saying anything to you. As they are done, one gives you <payment> credits.`

Reviewer: ```suggestion
		`Unfettered unload the materials you mined, without saying anything to you. As they are done, one gives you <payment>.`
```

> 		`The leader of the Unfettered unloading the materials comes to you, expressing deeps thanks, "This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks." He then hands you <payment> credits.`

Reviewer: ```suggestion
		`The leader of the Unfettered unloading the materials comes to you. "This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks." He then hands you <payment>.`
```

> 		`You hand over the minerals to the local Unfettered and they hand you <payment> credits.`

Reviewer: ```suggestion
		`You hand over the minerals to the local Unfettered and they hand you <payment>.`
```

> 		`As your crew is done unloading the materials, an Unfettered comes to you, looking humble and sincere. "Thank you, captain. We need these materials more than we would like to admit it, and the same goes for all help. If you have time, please consider doing other jobs, such as bringing food, for us." He then gives you <payment> credits.`

Reviewer: ```suggestion
		`As your crew is done unloading the materials, an Unfettered comes to you, looking humble and sincere. "Thank you, captain. We need these materials more than we would like to admit it, and the same goes for all help. If you have time, please consider doing other jobs, such as bringing food for us." He then gives you <payment>.`
```

mission "Unfettered Silicon Asteroid Mining"
	name `Gather silicon`
	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot, and then bring them back to <destination> for <payment>.`
	job
	repeat
	source "Darkcloak"
	to offer
		has "First Contact: Unfettered: declined"
		random < 60
	on complete
		payment 250000
		dialog phrase "unfettered mining payment dialog"
		outfit "Silicon" -50

mission "Unfettered Silver Asteroid Mining"
	name `Gather silver from asteroids`
	description `Mine and then collect 15 tons of silver from the asteroid field in Ehma Ti and then bring them back to <destination> for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	destination "Darkcloak"
	waypoint "Ehma Ti"
	to offer
		has "First Contact: Unfettered: declined"
		random < 60
	on complete
		payment 400000
		dialog phrase "unfettered mining payment dialog"
		outfit "Silver" -15

phrase "unfettered medicals payment dialog"
	word
		"The local health specialists are relieved that you have managed to bring the needed medical supplies back in time, and give you <payment> credits for it."

> 		"You have to wait a bit before being able to unload the medicals to the local hospital. All Unfettered there seem busy. After some time, a local worker gives you your <payment> credits and tells you to drop the cargo at a specific location."

Reviewer: ```suggestion
		"You have to wait a bit before being able to unload the medical supplies to the local hospital. All Unfettered there seem busy. After some time, a local worker gives you <payment> and tells you to drop the cargo at a specific location."
```

> 		`Once you arrive with the medical supplies, you are greeted by a local health specialist who seems to be desperate for them. "Not a moment too soon! If this was any later the consequences would have been dire." He adds, and then hands you <credits> credits.`

Reviewer: ```suggestion
		`Once you arrive with the medical supplies, you are greeted by a local health specialist who seems to be desperate for them. "Not a moment too soon! If this was any later the consequences would have been dire," he says, and then hands you <payment>.`
```

phrase "unfettered medicals pickup dialog"
	word
		`As usual, the Hai are more than happy to give you the medical supplies needed. "Tell our brothers and sisters to stay strong," one says while loading the supplies onto your ship.`
		`The Hai understand the urgency of the situation, and waste no time in talking. They load the supplies on your ship, and wish you a safe trip.`

> 		`There is no recitence to give you the medical supplies, it seems like the Hai are not really in need for them. "We prefer to make sure our brothers are not in need, our needs will come later," one says when he is done loading the supplies on your ship.`

Reviewer: ```suggestion
		`There is no recitence to give you the medical supplies. It seems like the Hai are not really in need for them. "We prefer to make sure our brothers and sisters are not in need. Our needs come later," one says when he is done loading the supplies on your ship.`
```

	on stopover
		dialog phrase "unfettered medicals pickup dialog"
	on complete
		payment 10000 4000
		dialog phrase "unfettered medicals payment dialog"

mission "Unfettered Big Medicals Rush"
	name `Medical supplies to <planet>`
	description `The Unfettered hospitals on <destination> are in need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
	job
	repeat
	deadline 15
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 1: offered"
		random < 80
	cargo "medicals" 35 2 .05
	stopover
		distance 5 7
		government "Hai"
		attributes "spaceport"
	on visit
		dialog phrase "generic missing stopover or cargo"
	on stopover
		dialog phrase "unfettered medicals pickup dialog"
	on complete
		payment 8000 3000
		dialog phrase "unfettered medicals payment dialog"

phrase "unfettered challenger defeated"
	word
		`You receive a message from the <npc>: "Did you cheat? If I learn you cheated, I will be the end of you, little monkey! Here is the <payment>, but I will want my money back if this was not a fair fight. Otherwise, you may have gained my respect."`

> 		`You receive a message from the <npc> : "Raaaah... Well, you have beaten me fairly, so I'll pay up... Here are the <payment> credits, captain. Next time I will not underestimate you anymore!"`

Reviewer: ```suggestion
		`You receive a message from the <npc>: "Raaaah... Well, you have beaten me fairly, so I'll pay up... Here are the <payment>, captain. Next time I will not underestimate you anymore!"`
```

> 		`You receive a message from the <npc> : "Nicely done! You seem to learn from our encounters well... But next time will be different, I assure you. For now, here are the agreed <payment> credits, captain. Hopefully it allows you to improve the <ship> before our next battle..."`

Reviewer: ```suggestion
		`You receive a message from the <npc>: "Nicely done! You seem to learn from our encounters well... But next time will be different, I assure you. For now, here are the agreed <payment>, captain. Hopefully it allows you to improve the <ship> before our next battle..."`
```

> 		`The <npc> hails you : "The fight is over! Stop shooting me or you'll blow holes into my ships! Here are the <payment> credits, let me be, now!"`

Reviewer: ```suggestion
		`The <npc> hails you: "The fight is over! Stop shooting me or you'll blow holes into my ships! Here are the <payment>, let me be now!"`
```

> 		`The <npc> hails you : "I surrender, <last>. You have indeed lived up to your reputation, I did not stand a chance in this battle. Still, a challenge is a challenge, here are your <payment> credits."`

Reviewer: ```suggestion
		`The <npc> hails you: "I surrender, <last>. You have indeed lived up to your reputation. I did not stand a chance in this battle. Still, a challenge is a challenge. Here are your <payment>."`
```

				fail "Unfettered Challenge [1]"

mission "Unfettered Challenge [2]"
	name `Defeat experienced Unfettered captain`
	description `An experienced Unfettered captain has challenged you into a duel. Disable his ship for <payment>.`
	job
	repeat
	source
		attributes "unfettered"
	to offer
		has "Unfettered Jump Drive 3: offered"
		has "Unfettered: Prove Your Strength"
		random < 10
	npc save disable
		government "Hai (Unfettered Challenge)"
		personality disables staying heroic target
		fleet
			names "hai"
			variant
				"Shield Beetle (Pulse)"
				"Sea Scorpion (Pulse)"
		dialog phrase "unfettered challenger defeated"
		conversation
			action
				payment 600000
				"reputation: Hai (Unfettered)" += 60
				fail "Unfettered Challenge [2]"

phrase "jump drive or stopover"
	word
		`You have a mission to bring a jump drive here on <planet>, from <stopovers>. You may have left your escort allocated to carrying the jump drive behind, in which case you better launch and wait for it.`

phrase "unfettered jump drive pickup dialog"
	word
		"You manage to land and retrieve the precious cargo without too much trouble. Do not rely on luck too long though - you should depart quickly!"

> 		"You find a few pirates in the vault but they are quickly disposed off, and you proceed to pick up the cargo. Do not wait too long, or reinforcements could arrive."

Reviewer: ```suggestion
		"You find a few pirates in the vault but they are quickly disposed of, and you proceed to pick up the cargo. Do not wait too long, or reinforcements could arrive."
```

		"It was easy, almost too easy to pick up the jump drives. Better depart before your luck runs out."

> 		"You receive a warning from the vault saying intruders came, but cannot find any trace of them. The cargo is swifly transferred, and you should probably get on your way now."

Reviewer: ```suggestion
		"You receive a warning from the vault saying that intruders came, but cannot find any trace of them. The cargo is swiftly transferred into your ship, and you should probably get on your way now."
```

		"The vault is secured and you can easily and safely pick up the jump drives."

phrase "unfettered jump drive payment dialog"
	word

> 		"The Unfettered are more than content with paying <payment> credits in exchange for your precious cargo."

Reviewer: ```suggestion
		"The Unfettered are more than content with paying <payment> in exchange for your precious cargo."
```

> 		"The Unfettered are in a rush to get this cargo, hoping to install it on their ships. They give you <payment> credits."

Reviewer: ```suggestion
		"The Unfettered are in a rush to get this cargo, hoping to install it on their ships. They give you <payment>."
```

> 		"Every time you bring jump drives back, it seems to be a party for the Unfettered. This time is no different, and they almost forget to give you the <payment> credits."

Reviewer: ```suggestion
		"Every time you bring jump drives back, it seems to be a cause for celebration for the Unfettered. This time is no different, and they almost forget to give you the <payment>."
```

		"Every time you bring jump drives back, it seems to be a cause for celebration for the Unfettered. This time is no different, and they almost forget to give you the <payment>."
		"Your cargo is unloaded, and the Unfettered give you <payment> for it. Curious about the source, you pay attention to a conversation between two members of the same crew and hear something about unfettered humans."
		"After unloading the cargo, you try your luck asking about who places these jump drives you're collecting, but, as usual, the Unfettered workers give you no information about it. Fortunately, they still give you <payment>."

mission "Get Jump Drives(1)"
	name "Collect a jump drive"
	description "Go to the pirate world of <stopovers> to recover a jump drive from a vault, then bring it to <destination> for <payment>."
	job
	repeat
	source
		attributes "unfettered"
	destination "Darkcloak"
	to offer
		random < 30
		has "Unfettered: Prove your strength 2: done"
	stopover
		distance 12 18
		government "Pirate"
	on stopover
		dialog phrase "unfettered jump drive pickup dialog"
		outfit "Jump Drive"
		"spawn jd fleet 1"++
	npc evade
		government "Hydra"
		to spawn

> 			# This means that if you have multiple missions for 1 jump drives you will get attacked by multiple fleets at the same time.

Author: Yeah mjltiple missions of the same category

> 			# This means that if you have multiple missions for 1 jump drives you will get attacked by multiple fleets at the same time.

Reviewer 5: ```suggestion
			# This means that if you have multiple missions for 1 jump drive each, you will get attacked by multiple fleets at the same time.
```

			branch "has respect"
				has "ships model (all): Modified Shield Beetle"
			choice
				`	(Take two tripulse shredders.)`
				`	(Take money instead.)`
					goto money
			label tripulse
			action
				outfit "Tripulse Shredder" 2
			`	The two shredders are loaded on your ship, in exchange for your jump drive.`
				goto trade

			label "has respect"
			`	They add that since you still have the modified shield beetle, you may access the special weaponry intended for it.`
			choice
				`	(Take two tripulse shredders.)`
					goto tripulse
				`	(Take 4 ionic blasters.)`
					to display
						"unfettered: prototypes exchanged" < 5
				`	(Take 1 ionic turret.)`
					to display
						"unfettered: prototypes exchanged" < 5
					goto turret
				`	(Take money instead.)`
					goto money
			action
				outfit "Ionic Blaster Prototype" 4
			`	The four blasters are loaded on your ship, in exchange for your jump drive.`
				goto prototype

			label turret
			action
				outfit "Ionic Turret Prototype"
			`	The turret is loaded on your ship, in exchange for your jump drive.`
			label prototype
			action
				"unfettered: prototypes exchanged" ++

> 			`	The shipyard authorities come to you with a grave look. "Unfortunately we cannot trade any more ionic weapons with you. Do note you can still exchange shredders for jump drives."`

Reviewer 5: In this context, jump drive is not capitalized.

> 			`	The shipyard authorities come to you with a grave look. "Unfortunately we cannot trade any more ionic weapons with you. Do note you can still exchange shredders for jump drives."`

Reviewer 6: This should be resolved than.

		random < 10
		"cargo space" > 80
	on complete
		payment 250000
		dialog phrase "unfettered mining payment dialog"
		outfit "Silicon" -100

mission "Unfettered Silver Asteroid Mining"
	job
	repeat
	name "Gather Silver from asteroids"
	description "Mine and collect 15 tons of silver, then bring the minerals back to <destination> for <payment>. Silver is available in the asteroid field in Ehma Ti."
	source
		attributes "unfettered"
		not planet "Warfeed"
	destination "Darkcloak"
	to offer
		random < 40
	on complete
		payment 450000
		dialog phrase "unfettered mining payment dialog"
		outfit "Silver" -15

mission "Unfettered Silver Asteroid Mining 2"
	job
	repeat
	name "Gather Silver from asteroids"
	description "Mine and collect 30 tons of silver, then bring the minerals to <destination> for <payment>. Silver is available in the asteroid field in Ehma Ti."
	source
		attributes "unfettered"
		not planet "Warfeed"
	destination "Darkcloak"
	to offer
		random < 10
		"cargo space" > 25
	on complete
		payment 1000000
		dialog phrase "unfettered mining payment dialog"
		outfit "Silver" -30

mission "Unfettered Lead Asteroid Mining"
	job
	repeat
	name "Gather Lead from asteroids"

> 	description "Mine and  collect 35 tons of lead, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."

Reviewer 5: ```suggestion
	description "Mine and collect 35 tons of lead, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."
```

				"reputation: Hai (Unfettered)" >?= 40

> 

Author: I think its 3 actually?

> 

Reviewer 6: Well, I have seen it vary. If it’s three, there is a lot that needs to change.

> 

Author: it's not enforced by the CI so I wouldn't worry too much about it honestly, I think I'm internally consistent at least

> 

Reviewer 6: Resolve it than.

# When outnumbered 2 to 1 at least, they may consider themselves unable to beat you and prefer to die standing than slaughtered.
mission "Proud Unfettered Boarded"
	boarding
	repeat
	minor
	to offer
		"flagship crew" > 120
		random < "flagship crew" / 8
	source
		government "Hai (Unfettered)"
		category "Heavy Warship"
	on offer
		conversation
			`As you board the <origin> you immediately notice something is wrong, the ship is starting to explode! The captain sends you a message. "This is not a fair fight. Better to die with my crew, standing, than risk being slaughtered by false, inferior, fettered humans! You will not be allowed to us this ship against our brothers and sisters, I will take you down with us!"`

> 				flee

Reviewer 2: I actually really like both how this in implemented and the response from the Unfettered Hai captain.

But, I'm missing how this mission destructs the ship?

Author: well the 'flee' makes the ship destroyed because this is a boarding mission

		outfit "Lead" -70

mission "Unfettered Titanium Asteroid Mining"
	job
	repeat
	name "Gather Titanium from asteroids"

> 	description "Mine and collect 50 tons of titanium, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."

Reviewer 5: Lead? What lead?

Author: ~~copy paste lead~~

	source
		attributes "unfettered"
		not planet "Warfeed"
	destination
		attributes "unfettered"
		not planet "Warfeed"
	to offer
		random < 40

> 		"cargo space" > 30

Reviewer 5: Should this be 30 or 50?

Author: it's a bit below what you have available like the others

			`	"To my knowledge, every Hai outfitters sells them, or almost, so you should not have any trouble finding them."`
				accept
	on complete
		outfit `"Basrem" Reverse Thruster` -1
		outfit `"Benga" Reverse Thruster` -1
		outfit `"Biroo" Reverse Thruster` -1
		payment 2000000
		dialog `The captain is really happy when you deliver the reverse engines to him, "I must try these on my Sea Scorpion and Shield Beetle to see what I can come up with, these look promising with long range weapons..." It looks like he could have kept on talking forever but he must have noticed you looking at him insistingly, "Oh, right, here is your payment of <payment>, I almost forgot!"`

mission "Unfettered Bring Batteries"
	job
	name "Bring Hai Valley Batteries"
	description "The Unfettered are running out of the biggest batteries, the manager of the outfitter wants you to bring Valley Batteries so they can still sell the right model to his client, before the <date>. They are willing to offer you <payment> for it."
	source
		attributes "unfettered"
	destination "Darkcloak"
	deadline 14
	to offer
		random < 5
		has "Unfettered Jump Drive 1: done"
	on offer
		conversation

> 			`As you accept the job, one of the workers of the outfitter comes to speak to you, "Do you have any questions regarding this job," he asks. "You do understand it must be done as fast as possible; as our new ships are in great needs of these."`

Reviewer 5: ```suggestion
			`As you accept the job, one of the workers of the outfitter comes to speak to you, "Do you have any questions regarding this job," he asks. "You do understand it must be done as fast as possible, as our new ships are in great needs of these."`
```

			label choose
			choice
				`	"Everything sounds clear."`
					accept
				`	"Why would you need such big batteries?"`
				`	"Where can I get these batteries?"`
					goto where
				`	"Can't you just make your own batteries?"`
					goto why

> 			`	"Well, as you have probably noticed, a lot of Hai use Ion Cannons in order to disable ships without doing too much damage to the crew. That in itself is a very good thing, but it also means that we need to actively work against it. Bigger batteries mean the guns will be less scrambled, and dimnishes the risk of being completely stranded in space."`

Reviewer 5: ```suggestion
			`	"Well, as you have probably noticed, a lot of Hai use Ion Cannons in order to disable ships without doing too much damage to the crew. That in itself is a very good thing, but it also means that we need to actively work against it. Bigger batteries mean the guns will be less scrambled, and diminishes the risk of being completely stranded in space."`
```

mission "Unfettered Bring Ion Cannons"
	job
	name "Bring Ion Cannons"
	description "Bring 2 Ion Cannons to the Unfettered on <destination>. They are willing to offer you <payment> for it."
	source
		attributes "unfettered"
	destination "Darkcloak"
	to offer
		random < 5
		has "Unfettered Jump Drive 1: done"
	on offer
		conversation
			`Seeing this peculiar request about Ion Cannons, which the Unfettered do have a bunch of, you go and look for a captain who made this job request, to ask them why they use those weapons. It does not take long to find them, and it seems like they are willing to answer your questions, at least partly, "I understand why you're wondering this, <first>... May I call you <first>? There are not many non pirates that come and supply us with jump drives, if at all, and that is how I know your name."`
			choice
				`	"Sure."`
					goto sure
				`	"I prefer we stick to a professional relation, call me <last>."`
			`	"Very well, not that it matters much anyway."`
			label sure

> 			`	"You see, if we use Ion it is not really for the same reason as the other Hai do, to hurt others of our kind as little as possible, although that is a noble cause indeed, it is not being noble that brings you food," she laughes a bit, and then comes back to a serious attitude, "No. If we do it, it is because it forces the Hai to use more batteries on their ships, as to not be disabled too easily, or defenceless because of weapon scrambling. So, we do not use ion weapons most of the time, but sometimes we surprise them with a big amount of them. Expecting this, the Hai are always forced to keep a good amount of batteries, and, thus, to have less guns, shields..."`

Reviewer 5: This paragraph could use some more work; the first part should definitely be rephrased.

		"Your cargo is unloaded, and the Unfettered give you <payment> for it. Curious about the source, you pay attention to a conversation between two members of the same crew and hear something about unfettered humans."
		"After unloading the cargo, you try your luck asking about who places these jump drives you're collecting, but, as usual, the Unfettered workers give you no information about it. Fortunately, they still give you <payment>."

mission "Unfettered Get Jump Drives(1)"
	job
	repeat
	name "Collect a jump drive"
	description "Go to the pirate world of <stopovers> to recover a jump drive from a vault, then bring it to <destination> for <payment>."
	source
		attributes "unfettered"
	stopover
		distance 12 18
		government "Pirate"
	clearance "When you tell the pirates you are here on behalf of the Unfettered, they immediately allow you to land without asking any questions."
	destination "Darkcloak"
	to offer
		random < 30
		has "Unfettered: Prove your strength 2: done"
	on stopover
		dialog phrase "unfettered jump drive pickup dialog"
		outfit "Jump Drive"

> 		"spawn jd fleet 1"++

Reviewer 5: ```suggestion
		"spawn jd fleet 1" ++
```
I think this one works? Idk, I see your version used in multiple missions here.

Author: ++ or space then ++ is the same yeah

	destination
		attributes "unfettered"
	to offer
		random < 20
		has "Unfettered: Materials Intercepted: done"
	npc board
		government "Pirate"
		personality disables plunders harvests staying
		system "Alcyone" "Misam"
		fleet
			names "pirate"
			variant
				"Bastion (Materials)"
			variant
				"Argosy (Materials)"
		dialog "This pirate ship indeed contains the materials you are looking for. You should bring them to <destination> to receive your payment."
	on visit
		dialog "You do not have the materials yet. Go look for them in eastern pirate space."
	on complete
		outfit "Materials" -100
		payment 1000000
		dialog phrase "unfettered cargo interception payment dialog"

mission "Unfettered Bring More Coolers"
	job
	repeat
	name "Bring 10 Liquid Nitrogen Coolers"
	description "The Unfettered want more nitrogen coolers in order to convert them into Zephyr Coolers. Bring 10 of them to <destination> for <payment>."
	source
		attributes "unfettered"
	destination "Firelode"
	to offer
		has "Unfettered: Solifuge 8: Useful Korath Scientists: done"
		random < 25
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		outfit "Liquid Nitrogen Cooler" -10

> 		dialog `You deliver the coolers to the usual contact of Mera, and he thanks you personally "You must have made a long trip to bing these here, for which I thank you. Here are <payment>. Whilst the price is not that high, your biggest reward is being able to buy the new Zephyr Coolings."`

Reviewer 5: ```suggestion
		dialog `You deliver the coolers to the usual contact of Mera, and he thanks you personally. "You must have made a long trip to bring these here, for which I thank you. Here are <payment>. Whilst the price is not that high, your biggest reward is being able to buy the new Zephyr Coolings."`
```

				goto choose
			label where
			`	"These batteries are pretty big, so I would expect them to mostly be in big outfitters, such as Markerplace. They should not be too hard to find."`
				goto choose
			label why
			`	"That is what we usually do, except that the need for those has increased recently, and we lack sufficient production to keep up with it. So we ask Hai Sympathizers and privateers like yourself to get them for us when we are in big need. Our navy must not seem weak or the Hai could see that as an opportunity to strike."`
				goto choose
	on complete
		outfit "Hai Valley Batteries" -1
		payment 500000
		dialog `You bring the batteries to the deck workers, who seem to be in a hurry to put them to use, "About time! Would it have been any later it would have probably been too late! Here are your <payment>."`

mission "Unfettered Bring Ion Cannons"

> 	job

Author: I dont think thats quite how it works, if you accept it and dont finish it it wont offer again, but until you actually accept it it should keep showing up? not sure though tbh

			choice
				`	"Thank you for the explanation, I will bring you these Ion Cannons."`
					goto money
				`	"And where can I find these Ion Cannons? Are they not restricted?"`
				`	"What did you mean, 'you hope I have a good raiding'?"`
					goto raiding
			`	"The Hai let anyone buy them, provided you have the money, of course."`
			label money
			action
				payment 1000000
				log "Minor People" "Binkeili" `An Unfettered that was interested in weapons dealing, and maybe something else, too.`
			`	"I will give you <payment> already, to make sure you can easily buy these Ion Cannons. Do not even think about trying to steal that money, captain. You will receive the rest later."`
				accept
			label raiding

> 			`	"Oh, that is just an expression we use, since we never know when we will see each other again, if at all. Too many do not come back from the raids, or even worse, they could come back with an empty cargo bay. Now, back to the task at hand!"`

Author: well she thinks they should not come back until they have something to bring yeah

> 			`	"Oh, that is just an expression we use, since we never know when we will see each other again, if at all. Too many do not come back from the raids, or even worse, they could come back with an empty cargo bay. Now, back to the task at hand!"`

Reviewer 6: This is resolved.

	destination "Darkcloak"
	to offer
		random < 30
		has "Unfettered: Prove your strength 2: done"
	on stopover
		dialog phrase "unfettered jump drive pickup dialog"
		outfit "Jump Drive"
		"spawn jd fleet 1" ++
	npc evade
		government "Cerberus"
		to spawn
			# This means that if you have multiple missions for 1 jump drive each, you will get attacked by multiple fleets at the same time.
			# I'd like to make it random, but that would mean they can spawn in other systems than the one you take the JD from.
			"spawn jd fleet 1" == "Get Jump Drives (1): offered"
		personality disables plunders vindictive waiting
		fleet "Small Northern Pirates" 2
	on visit
		dialog phrase "jump drive or stopover"
	on complete
		payment 500000
		outfit "Jump Drive" -1
		dialog phrase "unfettered jump drive payment dialog"

mission "Unfettered Get Jump Drives (2)"
	job
	repeat
	name "Collect 2 jump drives"
	description "Go to the pirate world of <stopovers> and recover 2 jump drives from a vault, then bring those to <destination> for <payment>."
	source
		attributes "unfettered"

> 	stopover

Author: I think it was fixed if that was the case? idk

> 	stopover

Reviewer 6: In 0.10.7, stopover through wormholes doesn’t work, so I’m not sure.

> 	stopover

Author: how so? I didnt find an issue looking at it real quick

phrase "unfettered cargo interception payment dialog"
	word
		"The Unfettered unloading your cargo of materials gives you <payment>. A significant amount for the cargo itself, but considering what it took to get it, the price is certainly fair."
		`An Unfettered unloading the materials comes to you, "It looks like the Elders are keeping their promises to rebuild, no matter the cost. Here are your <payment>."`
		`As you land, you are directed instead to bring the materials directly to their destination. It is a short trip, considering you are used to traveling in space. There, you receive <payment> for your troubles, before heading back to the spaceport.`
		`An Unfettered comes to you during the unloading of your cargo, "My deepest thanks, this will help in the repairs. We do not nearly produce enough materials to do it on our own." He then gives you <payment>.`
		"A spaceport worker gives you <payment> for the materials, without any further ceremony."
		`The leader of the Unfettered team that collects your cargo comes to you, "This is good, but we need more. Recently, an entire area collapsed because of lack of maintenance." He then hands you the chip with <payment>.`

> 

Author: I did it this way bc the phrase right above refers to this mission

mission "Unfettered Cargo Interception West"
	job
	repeat
	name `Find and bring the materials back`

> 	description `Find the <npc> that has stolen urban materials, in western pirate space, and bring them back to <destination> for <payment>. You may need to scan the cargo of ships to figure out which one has the materials.`

Author: they dont know about any southern pirates, I think

> 	description `Find the <npc> that has stolen urban materials, in western pirate space, and bring them back to <destination> for <payment>. You may need to scan the cargo of ships to figure out which one has the materials.`

Reviewer 6: Then this is resolved.

	source
		attributes "unfettered"
	destination
		attributes "unfettered"
	to offer
		random < 30
		has "<uhai> 7: Stolen Cargo: done"
	npc board
		government "Pirate"
		personality disables plunders harvests staying
		system "Almaaz" "Alnilam" "Arneb"
		fleet
			names "pirate"
			variant
				"Bastion (Materials)"
			variant
				"Argosy (Materials)"
		dialog "This pirate ship indeed contains the materials you are looking for. You should take them, and bring them to <destination> to receive your payment."
	on visit
		dialog `You do not have the materials yet. Go look for them in western pirate space.`
	on complete
		outfit "Building Materials" -100
		payment 1000000
		dialog phrase "unfettered cargo interception payment dialog"

mission "Unfettered Cargo Interception East"
	job
	repeat
	name `Find and bring the materials back`

> 	description `Find the <npc> that has stolen urban materials, in eastern pirate space, and bring them back to <destination> for <payment>. You may need to scan the cargo of ships to figure out which one has the materials.`

Author: honestly I'm not quite sure what to think so I'll just ask @tibetiroka what they think about it

		dialog "This pirate ship indeed contains the materials you are looking for. You should take them, and bring them to <destination> to receive your payment."
	on visit
		dialog "You do not have the materials yet. Go look for them in eastern pirate space."
	on complete
		outfit "Building Materials" -100
		payment 1000000
		dialog phrase "unfettered cargo interception payment dialog"

mission "Unfettered Bring More Coolers"
	job
	repeat
	name "Bring 10 Liquid Nitrogen Coolers"
	description "The Unfettered want more nitrogen coolers in order to convert them into Zephyr Coolers. Bring 10 of them to <destination> for <payment>."
	source
		attributes "unfettered"
	destination "Firelode"
	to offer
		has "unfettered: new cooling system"
		random < 20
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		outfit "Liquid Nitrogen Cooler" -10
		dialog `You deliver the coolers to the usual contact of Mera, and she thanks you personally. "You must have made a long trip to bring these here, for which I thank you. Here are <payment>. Whilst the price is not that high, your biggest reward is being continuously able to buy the new Zephyr Coolers."`
		payment 1250000

> 

Author: there being an empty space at the end of the file is a github convention I think

> 

Reviewer 6: Then resolve this.
